cmd/server.go

```diff
@@ -133,7 +134,7 @@ var Serve = &cobra.Command{
 		go eventHandler.ListenForEvents()
 
 		mgr, err := kopper.Manager(&kopper.ManagerOptions{
-			MetricsBindAddress: ":8081",
+			MetricsBindAddress: fmt.Sprintf(":%d", kopperPort),
```

Comment:
@yashmehrotra why do we need this? can we not leave this blank ?

Reply:
The default value for this is `8080` which collides with all our default ports, we can either give a port address or give `"0"` to disable it altogether

If we are disabling it, I suggest lets disable it in kopper